.eslintrc.json

```diff
@@ -13,6 +13,13 @@
     "react/forbid-prop-types": "off",
     "react/jsx-filename-extension": "off",
     "react/no-danger": "off",
-    "react/prefer-stateless-function": "off"
+    "react/prefer-stateless-function": "off",
+    "jsx-a11y/anchor-is-valid": [ "error", {
```

Comment:
not sure about this. @jenovs can you have a look at this please. linter is complaining <Link> requires an href attribute which doesn't make sense to me.